### PR TITLE
bump core to 2.71.3 in react-docs

### PR DESCRIPTION
**Pull request by redallen**
**What**: For updating Core's release notes in patternfly-org

### CODE REVIEW

File: packages/react-core/src/components/ApplicationLauncher/examples/ApplicationLauncher.md [hunk 1]
```diff
@@ -6,14 +6,15 @@ propComponents: ['ApplicationLauncher', 'ApplicationLauncherItem']
...
 
-Note: Application launcher is built on Dropdown, for extended API go to [`Dropdown`](/components/dropdown/) documentation.
-To add a tooltip, use the `tooltip` prop and optionally add more tooltip props by using `tooltipProps`. For more tooltip information go to [`Tooltip`](/components/tooltip/).
-
 import { ApplicationLauncher, ApplicationLauncherContent, ApplicationLauncherIcon, ApplicationLauncherText, ApplicationLauncherItem, ApplicationLauncherGroup, ApplicationLauncherSeparator, Text } from '@patternfly/react-core';
 import { HelpIcon, StarIcon } from '@patternfly/react-icons';
 import { Link } from '@reach/router';
 import pfIcon from './pf-logo-small.svg';
 
+Note: Application launcher is built on Dropdown, for extended API go to [Dropdown](../../components/dropdown/) documentation.
```
Comment (evwilkin) [inline packages/react-core/src/components/ApplicationLauncher/examples/ApplicationLauncher.md:14]: I think you've gone 1 level too high here and for Tooltip below
Screen Shot 2020-04-01 at 9 31 25 AM

Comment (redallen) [reply packages/react-core/src/components/ApplicationLauncher/examples/ApplicationLauncher.md]: What matters is that it works as as relative path in the docs... I just pushed an update that I think will work.

File: packages/react-docs/RELEASE-NOTES.md [hunk 2]
```diff
@@ -37,18 +37,21 @@ Packages released:
...
+- **Docs:** 
+  - Release docs to NPM for patternfly-org ([#3941](https://github.com/patternfly/patternfly-react/pull/3941))
+- **Build:**
+  - Created unified react-core dist ([#3971](https://github.com/patternfly/patternfly-react/pull/3971))
+  - In-house focusTrap ([#3967](https://github.com/patternfly/patternfly-react/pull/3967))
+  - Remove unused packages from repo ([#3916](https://github.com/patternfly/patternfly-react/pull/3916))
+  - Create variables by file in react-tokens ([#3896](https://github.com/patternfly/patternfly-react/pull/3896))
+- **Types:**
+  - Remove a few usages of prop-types ([#3968](https://github.com/patternfly/patternfly-react/pull/3968))
+  - Fix tippy types, remove copyTS ([#3940](https://github.com/patternfly/patternfly-react/pull/3940))
```
Comment (tlabaj) [inline packages/react-docs/RELEASE-NOTES.md:54]: removed

File: packages/react-docs/RELEASE-NOTES.md [hunk 3]
```diff
@@ -37,18 +37,21 @@ Packages released:
...
-- Remove unused packages from repo ([#3916](https://github.com/patternfly/patternfly-react/pull/3916))
+- **Docs:** 
+  - Release docs to NPM for patternfly-org ([#3941](https://github.com/patternfly/patternfly-react/pull/3941))
+- **Build:**
+  - Created unified react-core dist ([#3971](https://github.com/patternfly/patternfly-react/pull/3971))
+  - In-house focusTrap ([#3967](https://github.com/patternfly/patternfly-react/pull/3967))
+  - Remove unused packages from repo ([#3916](https://github.com/patternfly/patternfly-react/pull/3916))
+  - Create variables by file in react-tokens ([#3896](https://github.com/patternfly/patternfly-react/pull/3896))
+- **Types:**
+  - Remove a few usages of prop-types ([#3968](https://github.com/patternfly/patternfly-react/pull/3968))
```
Comment (tlabaj) [inline packages/react-docs/RELEASE-NOTES.md:53]: Removed

File: packages/react-docs/RELEASE-NOTES.md [hunk 4]
```diff
@@ -37,18 +37,21 @@ Packages released:
...
-- Remove a few usages of prop-types ([#3968](https://github.com/patternfly/patternfly-react/pull/3968))
-- Release docs to NPM for patternfly-org ([#3941](https://github.com/patternfly/patternfly-react/pull/3941))
-- Fix tippy types, remove copyTS ([#3940](https://github.com/patternfly/patternfly-react/pull/3940))
-- Create variables by file in react-tokens ([#3896](https://github.com/patternfly/patternfly-react/pull/3896))
-- Remove unused packages from repo ([#3916](https://github.com/patternfly/patternfly-react/pull/3916))
+- **Docs:** 
+  - Release docs to NPM for patternfly-org ([#3941](https://github.com/patternfly/patternfly-react/pull/3941))
+- **Build:**
+  - Created unified react-core dist ([#3971](https://github.com/patternfly/patternfly-react/pull/3971))
+  - In-house focusTrap ([#3967](https://github.com/patternfly/patternfly-react/pull/3967))
```
Comment (tlabaj) [inline packages/react-docs/RELEASE-NOTES.md:49]: maybe "Added In-house focusTrap"

File: packages/react-docs/RELEASE-NOTES.md (hunk 2 above)
Comment (tlabaj) [inline packages/react-docs/RELEASE-NOTES.md:54]: Fixed

File: packages/react-docs/RELEASE-NOTES.md [hunk 5]
```diff
@@ -37,18 +37,21 @@ Packages released:
...
+  - Remove unused packages from repo ([#3916](https://github.com/patternfly/patternfly-react/pull/3916))
```
Comment (tlabaj) [inline packages/react-docs/RELEASE-NOTES.md:50]: Removed

Comment (redallen) [reply packages/react-docs/RELEASE-NOTES.md:50]: I accidentally committed this file. I'll have to update it in the next PR once I know the new version numbers that need to be released.

Comment (redallen) [reply packages/react-docs/RELEASE-NOTES.md:50]: For now I think the changes it makes are okay... can you approve? It's the thing I'm going to work on right after this gets merged and there's newly released versions. See my message in Slack for the release timeline.

File: packages/react-docs/RELEASE-NOTES.md [hunk 6]
```diff
@@ -21,7 +21,7 @@ Packages released:
 - **About modal:**
   - Allow custom aria-label for the close button ([#3877](https://github.com/patternfly/patternfly-react/pull/3877))
```
Comment (rachael-phillips) [inline packages/react-docs/RELEASE-NOTES.md:22]: Allowed in past tense instead of allow :)

File: packages/react-docs/RELEASE-NOTES.md [hunk 7]
```diff
@@ -37,18 +37,21 @@ Packages released:
   - Allow count badge to be hidden in checkbox select ([#3976](https://github.com/patternfly/patternfly-react/pull/3976))
```
Comment (rachael-phillips) [inline packages/react-docs/RELEASE-NOTES.md]: Allowed in past tense instead of allow :)

File: packages/react-docs/RELEASE-NOTES.md (hunk 4 above)
Comment (rachael-phillips) [reply packages/react-docs/RELEASE-NOTES.md]: That makes sense to me!

File: packages/react-docs/RELEASE-NOTES.md [hunk 8]
```diff
@@ -37,18 +37,21 @@ Packages released:
...
-- Fix tippy types, remove copyTS ([#3940](https://github.com/patternfly/patternfly-react/pull/3940))
-- Create variables by file in react-tokens ([#3896](https://github.com/patternfly/patternfly-react/pull/3896))
-- Remove unused packages from repo ([#3916](https://github.com/patternfly/patternfly-react/pull/3916))
+- **Docs:** 
+  - Release docs to NPM for patternfly-org ([#3941](https://github.com/patternfly/patternfly-react/pull/3941))
+- **Build:**
+  - Created unified react-core dist ([#3971](https://github.com/patternfly/patternfly-react/pull/3971))
+  - In-house focusTrap ([#3967](https://github.com/patternfly/patternfly-react/pull/3967))
+  - Remove unused packages from repo ([#3916](https://github.com/patternfly/patternfly-react/pull/3916))
+  - Create variables by file in react-tokens ([#3896](https://github.com/patternfly/patternfly-react/pull/3896))
```
Comment (rachael-phillips) [inline packages/react-docs/RELEASE-NOTES.md]: Created

File: packages/react-docs/RELEASE-NOTES.md (hunk 6 above)
Comment (rachael-phillips) [inline packages/react-docs/RELEASE-NOTES.md]: Allowed

**Comment by redallen** [comment]
@tlabaj @rachael-phillips I did not mean to commit release note changes. I have reverted them and they will be updated in the next PR once I know the right version numbers from releasing this PR.

File: packages/react-docs/RELEASE-NOTES.md (hunk 7 above)
Comment (rachael-phillips) [inline packages/react-docs/RELEASE-NOTES.md]: Allowed

File: packages/react-docs/RELEASE-NOTES.md [hunk 9]
```diff
@@ -37,18 +37,21 @@ Packages released:
...
 ### Other
-- Create unified react-core dist ([#3971](https://github.com/patternfly/patternfly-react/pull/3971))
-- In-house focusTrap ([#3967](https://github.com/patternfly/patternfly-react/pull/3967))
-- Remove a few usages of prop-types ([#3968](https://github.com/patternfly/patternfly-react/pull/3968))
-- Release docs to NPM for patternfly-org ([#3941](https://github.com/patternfly/patternfly-react/pull/3941))
-- Fix tippy types, remove copyTS ([#3940](https://github.com/patternfly/patternfly-react/pull/3940))
-- Create variables by file in react-tokens ([#3896](https://github.com/patternfly/patternfly-react/pull/3896))
-- Remove unused packages from repo ([#3916](https://github.com/patternfly/patternfly-react/pull/3916))
+- **Docs:** 
+  - Release docs to NPM for patternfly-org ([#3941](https://github.com/patternfly/patternfly-react/pull/3941))
```
Comment (rachael-phillips) [inline packages/react-docs/RELEASE-NOTES.md]: Moved or Added in front oh phrase

File: packages/react-core/src/components/ApplicationLauncher/examples/ApplicationLauncher.md (hunk 1 above)
Comment (redallen) [reply packages/react-core/src/components/ApplicationLauncher/examples/ApplicationLauncher.md]: Pushed another update to use the absolute path. Hopefully it builds in patternfly-org with the path prefix okay!

**Pull request merged**